nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -189,20 +189,15 @@ let
 
       ${optionalString (cfg.recommendedGzipSettings) ''
         gzip on;
-        gzip_proxied any;
-        gzip_comp_level 5;
-        gzip_types
-          application/atom+xml
-          application/javascript
-          application/json
-          application/xml
-          application/xml+rss
-          image/svg+xml
-          text/css
-          text/javascript
-          text/plain
-          text/xml;
+        gzip_static on;
         gzip_vary on;
+        gzip_disable "MSIE [1-6]\.";
```

Comment:
Do we still need to support ie6? I am sure it won't be compatible with most modern websites or even our TLS settings anyway.

Reply:
I am against adding Internet Explorer 6 compatibility at this point.

Reply:
Ok, i will removed this parameter.